stdio/Cargo.toml

```diff
@@ -11,12 +11,12 @@ version = "14.0.1"
 
 [dependencies]
 futures = "0.1.23"
-jsonrpc-core = { version = "14.0", path = "../core" }
```

Comment:
Please revert this change, we want the local crates to use local dependencies, not go to crates.io every time.